google-finance/src/components/Nav/ActiveStock/ActiveStockContainer.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import StockList from './StockList';
+import styled from 'styled-components';
+import theme from '../../../styles/theme';
+
+const ActiveStockContainer = () => (
+  <div>
```

Comment:
얘는 
```suggestion
  <>
  ...
  </>
```
일케하면 좋을거같아용!

Reply:
오잉 아니다 아예 그냥
```suggestion
   <St.ActiveStockContainer>
      <St.NavTitle>가장 거래가 활발한 주식</St.NavTitle>
      <StockList />
    </St.ActiveStockContainer>
```
위만 남기는게 좋겠닷!